api/SubtleCrypto.json

```diff
@@ -12,7 +12,8 @@
             "version_added": "37"
           },
           "deno": {
-            "version_added": "1.11"
+            "version_added": "1.11",
+            "partial_implementation": true
```

Comment:
I had set it to partial because some of the descendants are `"version_added": false`. That doesn't make much sense though as you said, so I have removed the `"partial_implementation"` from the interface. It is still present on some descendants, but with `"notes"`.